docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_sqlite_data.md

```diff
@@ -23,33 +23,26 @@ In this guide we will demonstrate how to connect Great Expectations to data in a
 
 ### 1. Import GX and instantiate a Data Context
 
-```python title="Python code"
-import great_expectations as gx
-
-context = gx.get_context()
+```python name="tests/integration/docusaurus/connecting_to_your_data/how_to_connect_to_sqlite_data.py import"
```

Comment:
@TrangPham We still use `<ImportGxAndInstantiateADataContext />` -- it is old style, but the benefit is that it contains some helpful text.  Thanks.

Reply:
@TrangPham And this can be imported (please see other documents that import this component.  Thanks.